sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -146,6 +150,15 @@ async def load(*args, **kwargs) -> "AzureAppConfigurationProvider":
     if (endpoint or credential) and connection_string:
         raise ValueError("Please pass either endpoint and credential, or a connection string.")
 
+    # Removing use of AzureAppConfigurationKeyVaultOptions
+    if key_vault_options:
+        if "key_vault_credentials" not in kwargs:
```

Comment:
This should be singular. I'm also wondering if there's a compelling reason to not use `keyvault` rather than `key_vault` which just looks a bit more awkward.... 
```suggestion
        if "keyvault_credential" not in kwargs:
```

Reply:
Also - could we update these new settings in the docstring?